cmake/FindOpusFile.cmake

```diff
@@ -1,20 +1,31 @@
 include(FindPackageHandleStandardArgs)
 
+find_package(PkgConfig QUIET)
+pkg_check_modules(PC_OPUSFILE QUIET opusfile)
```

Comment:
I think this must be executed conditionally if pkg-config is not installed.

Reply:
It is safe to call even when the pkg-config executable is not found.

Internally, the modules guards the call with `if(PKG_CONFIG_EXECUTABLE)`, setting `PKG_CONFIG_EXECUTABLE` manually to `PKG_CONFIG_EXECUTABLE-NOTFOUND` does not break configuration, instead, it skips searching and sets all the result variables to an empty string.

Reply:
Yup. [You are right](https://github.com/Kitware/CMake/blob/v3.0.0/Modules/FindPkgConfig.cmake#L263).